### DISCUSSION TITLE
The cables.gl website keeps loading?

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#354] [Answers & Questions] [answered]
The website isn't loading for me since yesterday. At times, it barely manages menu text with a black background. Console log screenshot attached:
[https://postimg.cc/d79XmJbG](url)

**Comment by monolithMktg** [comment]
It's working well on my mobile. Any help why it isn't opening on a desktop browser?

**Comment by pandrr** [comment]
- screenshot is deleted
- any errors in dev tools console ?
- which browser ?
- try disabling extensions?

**Reply by monolithMktg** [reply]
Sorry for the screenshot delete pandrr. It was supposed to be expired after a month.

Anyway, **quick update,** it is loading, now no more errors in console, but it is is very sluggish. and also not getting 100% loaded. It has its own mind. Sometimes the options on hovering 'Create' open, sometimes they don't. But thumbnails do not display on quick begin patches apart from Custom Shader as can be seen in the screenshot below. Happens in every browser: Firefox, Chrome, and Edge. No extensions at all.

### Hero Section Masonry
![image](https://user-images.githubusercontent.com/94358787/192092723-eed002dc-9701-424c-976d-b137a27f8897.png)


### Patch images not showing
![image](https://user-images.githubusercontent.com/94358787/192092641-744ebee9-54ec-4608-bd82-20d4a1e0fda7.png)

### Occassional Console Log error:
![image](https://user-images.githubusercontent.com/94358787/192093336-cd391d5b-bdd6-41ba-b3ed-63039ffcb589.png)

**Comment by monolithMktg** [comment, marked as answer]
Mobile hotspot connection is loading the site well on desktop. So the problem is my wifi router. Sigh. Why the heck it limited the connection to Cables Gl is beyond me. I'll investigate further thanks. But if someone knows something about it please share. :(

**Reply by pandrr** [reply]
i would like to see the network tab ? is it canceling those files because of timeouts ?

**Reply by monolithMktg** [reply]
Thank you, @pandrr . But I formatted my Laptop. Now it works fine. I have no idea what was causing it. Maybe the Anti-virus or a windows update.
If that happens again, I shall share the network tab info with you. Thanks again. 👍
